### DISCUSSION TITLE
Disable Markdown preview on double click

### DISCUSSION THREAD

**Discussion by eekhof** [microsoft/vscode-jupyter#12159] [Questions and Answers] [answered]
Hello,

I would like to disable the feature of a preview-window with rendered Markdown popping up to the right of my notebook each time I double click on a Markdown cell.

![image](https://user-images.githubusercontent.com/105216949/203755176-8f967329-27b4-47d1-824b-eed66ae3c83f.png)
(With preview-window I mean the window/tabs on the right)

I was unable to find anything regarding this question, I am sorry if I overread something. I would be very grateful for help.

All the best, many thanks,
Lucas

**Comment by amunger** [comment, marked as answer]
This doesn't sound like anything we've implemented, and I can't get this to repro on my machine - double clicking a rendered markdown cell goes to edit mode, and double clicking within edit mode does nothing. 
Can you try to run the `Help: Start Extension Bisect` command to check if it's a different extension that is causing this behavior?

**Reply by eekhof** [reply]
Thank you for your help, I think you were right - I just changed the setting from the markdown extension `markdown.extension.preview.autoShowPreviewToSide` to false, and now the behavior is just as I would like it, and as you describe it.
